src/lib/core/components/Form/types/input.ts

```diff
@@ -7,6 +7,7 @@ import {FieldRenderProps, FieldValue, LayoutType} from './';
 export type InputProps<Value extends FieldValue, SpecType extends Spec> = {
     spec: SpecType;
     name: string;
+    inputProps?: SpecType['viewSpec']['inputProps'];
```

Comment:
let's also make this type universal with the generic, and it will be right to create same logic for other types, i mean `IndependentInputProps, InputType, IndependentInputType`

Reply:
If we do so, we need to add generic for common `Spec` type, do we really need to duplicate generic logic for base type? We already have generic for string/array/bool/object input and layout props